pkg/webui/components/button/index.js

```diff
@@ -49,6 +49,7 @@ class Button extends React.PureComponent {
       className,
       onClick,
       error,
+      link,
       ...rest
```

Comment:
i would also prefer filtering link and button related props. there is absolutely no reason to pass, for example, the `onClick` prop to a link. see `L73`